orchestrator/gbt/src/keys/mod.rs

```diff
@@ -1,12 +1,13 @@
 pub mod register_orchestrator_address;
 
```

Comment:
you can clean up these empty lines

Reply:
We could also use the `One` setting https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#group_imports, but I have always personally used `StdExternalCrate`.

Reply:
I presume you are ok